### PR TITLE
Feature: JSON config

### PR DESCRIPTION
**Pull request by anned20**

### CODE REVIEW

**Comment by anned20** [comment]
@vimsence/maintainers, can you check this?

File: python/vimsence.py [hunk 1]
```diff
@@ -33,12 +36,11 @@
 if vim.eval('exists("g:vimsence_client_id")') == '1':
     client_id = vim.eval('g:vimsence_client_id')
 
+with open(os.path.join(vim.eval('s:plugin_root_dir'), '..', 'vimsence.json'), 'r') as config_file:
+    config = json.load(config_file)
+
 # Contains which files has thumbnails.
```
Comment (hugolgst) [inline python/vimsence.py:42]: Could you update the comment to explain that we're retrieving the data from the configuration file?

File: vimsence.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+filetypes:
```
Comment (hugolgst) [inline vimsence.yaml:1]: Is this file used? Why having a .json and a .yaml ?

Comment (anned20) [reply vimsence.yaml:1]: Yep, you're right, I'll remove it.

**Review by hugolgst — APPROVED** [review]

**Pull request merged**